### DISCUSSION TITLE
Correct local time

### DISCUSSION THREAD

**Discussion by beneva52** [tbnobody/OpenDTU#483] [Ideas]
Hello,
Could you please change line 665 in HomeView.vue to use local time according to selected language?
Thanks in advance,
Ben

./webapp/src/views/HomeView.vue:
665: const userLocale = **localStorage.getItem("locale") ||** navigator.language;

**Comment by tbnobody** [comment]
Hi,

thank you for the hint.

`const userLocale = this.$i18n.locale; `
would maybe also make sense...

Will doublecheck what vue-i18n provides for formatting date/time

**Comment by tbnobody** [comment]
It's implemented in ef8c3a3d4267e7a62a634792c461e0f7c913719b

**Reply by beneva52** [reply]
Thanks! Using vue-i18n is probably more robust.
Could you please also modify "number.ts" according to the following code?

```
export const formatNumber = (num: number, digits: number): string => {
    return new Intl.NumberFormat(
        localStorage.getItem("locale") || navigator.language, { minimumFractionDigits: digits, maximumFractionDigits: digits }
    ).format(num);
}
```
